gempy/plot/plot.py

```diff
@@ -217,7 +217,8 @@ def plot_predef_sections(model, show_traces=True, show_data=False, section_names
 
 
 def plot_section(model, cell_number=13, block=None, direction="y", interpolation='none',
-                 show_data=False, show_faults=True, show_topo = False,  block_type=None, ve=1, **kwargs):
+                 show_data=False, show_faults=True, show_topo=False, show_legend=False, 
```

Comment:
Do you think that by default is better False than True?

Reply:
hmm, good question! Thinking about it, default `True` might be better, as people will initially want to have it I think..
```suggestion
                 show_data=False, show_faults=True, show_topo=False, show_legend=True, 
```